content/tags/theme_tags.md

```diff
@@ -0,0 +1,281 @@
+# Theme Tags
+
+Theme Tags have various functions, including:
+
+* Outputting template-specific HTML markup
+* Telling the theme which layout and snippets to use
+* Splitting a returned array into multiple pages.
+
+<br>
+
+1. TOC
+{:toc}
+
+## comment
+
+Allows you to leave un-rendered code inside a Liquid template. Any text within the opening and closing `comment` blocks will not be output, and any Liquid code within will not be executed.
+
+
+#### Input
+
+~~~ liquid
+  Best for me is {% comment %}super{% endcomment %} BookingSync.
+~~~
+
+#### Output
+
+~~~
+  Best for me is BookingSync.
+~~~
+
+## include
+
+Inserts a **snippet** from the snippets folder of a theme.
+
+Note that the *.liquid* extension is omitted in the above code.
+When a snippet is included, the code inside it will have access to the variables within its parent template.
+
+### Including multiple variables in a snippet
+
+There are two ways to include multiple variables in a snippet. You can assign and include them on different lines:
+
+~~~ liquid
+  {% assign snippet_variable = 'this is it' %}
+  {% assign snippet_variable_two = 'this is also it' %}
+  {% include 'snippet' %}
+~~~
+
+Or you can consolidate them into one line of code:
+
+~~~ liquid
+  {% include 'snippet', snippet_variable: 'this is it', snippet_variable_two: 'this is also it' %}
+~~~
+
+> ### parameters `include`
+>
+> ### *with*
+>
+> The `with` parameter assigns a value to a variable inside a snippet that shares the same name as the snippet.
+>
+> For example, we can have a snippet named **color.liquid** which contains the following:
+>
+> ~~~ liquid
+>   color: '{{ color }}'
+>   shape: '{{ shape }}'
+> ~~~
+>
+> Within **theme.liquid**, we can include the **color.liquid** snippet as follows:
+>
+> ~~~ liquid
+>   {% assign shape = 'circle' %}
+>   {% include 'color' %}
+>   {% include 'color' with 'red' %}
+>   {% include 'color' with 'blue' %}
+>   {% assign shape = 'square' %}
+>   {% include 'color' with 'red' %}
+> ~~~
+>
+> The output will be:
+>
+> ~~~ css
+>   color: shape: 'circle'
+>   color: 'red' shape: 'circle'
+>   color: 'blue' shape: 'circle'
+>   color: 'red' shape: 'square'
+> ~~~
+
+## form
+
+Creates an HTML `<form>` element with all the necessary attributes (action, id, etc.) and `<input>` to submit the form successfully.
```

Comment:
not sure here, the examples seem to be abstract to the website app, maybe it could be refactored to show some real bsa-website usage?